src/content/structured/get-started/custom-theme.mdx

```diff
@@ -3,24 +3,29 @@ path: "/get-started/install-components/custom-theme"
 
 navPriority: 9
 
-date: "2024-02-05"
+date: "2024-12-02"
 
 title: "Custom theme"
 
-subTitle: "How to change the theme colour of some components."
+subTitle: "How to change the colours of some components."
 
 contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/get-started/custom-theme.mdx"
 ---
 
+import { useState } from "react";
+import { IcAlert, IcButton, IcTheme } from "@ukic/react";
+
 ## Customising component colours
 
-Some components can have their theme colour changed by using either of the methods below. When the theme colour is changed, components will automatically update.
```

Comment:
Think it would be good to have a paragraph above this section explaining the difference between theming and branding in the ICDS

Reply:
I'm adding a couple of lines in. Let me know what you think